app/models/saved_claim/education_benefits/va_1995.rb

```diff
@@ -2,4 +2,46 @@
 
 class SavedClaim::EducationBenefits::VA1995 < SavedClaim::EducationBenefits
   add_form_and_validation('22-1995')
+
+  # Pulled from https://github.com/department-of-veterans-affairs/vets-website/src/applications/edu-benefits/utils/helpers.jsx#L100
+  # & fixtures at https://github.com/department-of-veterans-affairs/vets-website/src/applications/edu-benefits/1995/tests/e2e/
```

Comment:
Shouldn't it be the helpers file and this file? https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/edu-benefits/utils/labels.jsx